docs/_documentations/workingwithtemplates.md

```diff
@@ -90,9 +90,13 @@ Add your template sources to Codewind with the **Template Source Manager**.
 
 1. Add the template.
    - **VS Code:** In the **Template Source Manager**, click **Add New**.
-   - **Eclipse:** Right-click the connection in the Codewind Explorer view and select **Manage Template Sources...**. After the **Manage Template Sources** wizard appears, click **Add...**.
-   - **IntelliJ:** Right-click the connection in the Codewind Explorer view and select **Manage Template Sources...**. After the **Manage Template Sources** wizard appears, select **My Private Template Repository** and click **OK**.
-2. Enter the URL and any other information.
+   - **Eclipse and IntelliJ:**
+     - Right-click the connection in the Codewind Explorer view and select **Manage Template Sources...**.
```

Comment:
I missed this.  In Eclipse, it is: "Codewind Explorer" view.  In IntelliJ, it is: "Codewind" view.